src/components/UserCard.tsx

```diff
@@ -1,37 +1,51 @@
-import Image from 'next/image'
-
-// import sealife from '../../public/sealife-images/sealife_250px_001.webp'
+import JellyFishOutlineIcon from './icons/JellyFishOutlineIcon'
 
 type CardProps = {
 	name: string
-	storyPoints: number
+	storyPoints: string
```

Comment:
suggestion: This ain't a string, this is a StoryPoints types, where it can be 0,1,2,3,5,etc.

Reply:
It also can be a '?' if that is an allowed storypoint value, which I have allowed. So pass them all as a string makes it easier.